src/chain/index.js

```diff
@@ -37,13 +37,32 @@ import { required } from '@stamp/required'
  * @return {Object} Chain instance
  */
 const Chain = stampit({
-  deepProps: { Ae: { defaults: { waitMined: true, verify: true } } }
+  deepProps: {
+    Ae: {
+      defaults: {
+        waitMined: true,
+        verify: true,
+        _expectedMineRate: 180000,
+        _microBlockCycle: 3000,
+        _maxPollInterval: 5000
+      }
+    }
+  },
+  methods: {
+    _getPollInterval (type) {
+      const base = {
+        block: this.Ae.defaults._expectedMineRate,
+        microblock: this.Ae.defaults._microBlockCycle
+      }[type]
+      if (!base) throw new Error(`Unknown poll type: ${type}`)
+      return Math.min(base / 3, this.Ae.defaults._maxPollInterval)
```

Comment:
Shouldn't the methods and config above either moved to a config then applied here or be somehow the defaults? I don't understand why this is specified here.

Reply:
these parameters are not configurable for the developers. `getPollInterval` method is used in case the option `interval` is not specified. 

e.g.

```js
function pollForQueries (
  oracleId,
  onQuery,
  { interval = this._getPollInterval('microblock') } = {}
) {
  const knownQueryIds = new Set()
  ....
  }
  ```
  
  Value of the defaults could be moved to `schema.js` file.

Reply:
`schema.js` is the wrong place for this IMO as it's mainly (or even exclusively) used for tx related config values.

Reply:
Okay. Makes sense. I remember having the same discussing with Denis.